edgraph/access_ee_test.go

```diff
@@ -49,17 +49,43 @@ func sliceCompare(x, y []string) bool {
 
 	return true
 }
+func TestPublicKey(t *testing.T) {
+	tokenString := "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3NzU3OTg5MDQsImdyb3VwcyI6WyI3MDEiLCI3MDIiXSwibmFtZXNwYWNlIjoxMjM0NTY3ODkwLCJ1c2VyaWQiOiJ1c2VyMSJ9.kVWrSQgWatrKToaS3DDxAqIx2QGx_6ubkuyCyTTLF8AjkaHLN2nDa-HhW5G_aGxh4-l_X6gCsyay2wUs9cn2R6VD53AUrEdEtF0davsTX0Q1ar-RRo8ShTHj0ufgLn7zRGC7grbZrNZJi1RtDWGDRKzzt67nG-a2CooyP40s-1I--g9qQJoWu69moWiDH6ac2tLo7-wBRmgtWn-SgXE-nJYVDOUcaNWSQBwEX5Pe5L2OUYiusCxnlBhpSSREGYcnXdx9jPKEhfl9atg9QkJSwGyRrCtGY_6gA8tvSZdB2rd6pgXamoxxBadEr_eURg0OeJAJ8wf_IlpS6boOhKNt5Q"
```

Comment:
we could and maybe we should use openssl to generate these keys on the fly. This way we will make sure that we are always compatible with openssl, which is the industry standard.